airbyte-integrations/connector-templates/source-generic/NEW_SOURCE_CHECKLIST.md.hbs

```diff
@@ -0,0 +1,15 @@
+# New generic source checklist
+
+This is an autogenerated file describing the steps needed to implement a new Airbyte source.
```

Comment:
made this one much shorter than the average checklist. I think from now on we should put code-specific action items as TODOs in the code, and cover more conceptual topics in the docs. That's the idea anyways.